documentation/starter-kits/GOVERNANCE/GOVERNANCE.md

```diff
@@ -0,0 +1,117 @@
+
+- [INSERT LINK TO DEPENDENT PROJECT REPOSITORIES]
+
+
+### Decision Making Process
+
+Any community member can create a GitHub issue or comment asking the TSC to review something. Prior to implementing a substantial contribution, the design of that contribution should be reviewed by at least one member of the TSC. If consensus-seeking fails during the review of a pull request or in design discussions, the issue will be addressed by the TSC to make a determination on direction. TSC members will meet regularly and will keep track of decisions made when consensus was not met. 
+
+The TSC can nominate new members at any time. Candidates for membership tend to be collaborators who have shown great dedication to the project and/or experts in a particular domain or project component. TSC members are expected to be active contributors or members who have made significant contributions within the past 12 months. 
+
+## Project Management Committee 
+
+The Project Management Committee (PMC) is made up of sponsor organization representatives (i.e. those providing funding to the project) and key stakeholders who rely or plan to rely on [INSERT PROJECT NAME] to meet a critical need. The PMC has the following primary responsibilities
+
+- Maintaining the overall project roadmap
+- Determining project requirements and commitments to sponsors and stakeholders
+- Assessing available resources and allocating them across the project
+- Monitoring and reporting on progress against the roadmap 
+- On-boarding new sponsors and stakeholders
+- Overall project governance (including this policy)
+- Addressing any legal considerations
+
+### PMC Committee Members
+- [INSERT MEMBER NAME] ([username1]([INSERT LINK TO USERNAME]), [INSERT ORG ASSOCIATION]
+- [INSERT MEMBER NAME] ([username1]([INSERT LINK TO USERNAME]), [INSERT ORG ASSOCIATION]
+
+<details>
+
+<summary>Emeriti</summary>
+
+### PMC Emeriti
+- [INSERT MEMBER NAME] ([username1]([INSERT LINK TO USERNAME]), [INSERT ORG ASSOCIATION]
+
+</details>   
+
+### Scope
+
+The PMC has management authority over the same projects over which the TSC has technical authority.   
+
+### Decision Making Process
+
+The PMC will consist of a product manager and additional representative from sponsors and key stakeholders. The PMC or sponsoring organizations can nominate new members at any time. Care should be taken not to include too many members from a single stakeholder project or organization.
+
+Monthly stakeholder meetings are held to discuss current project status and propose changes to the project roadmap. If stakeholder representatives and sponsors concur with these proposals during the meeting, they will be immediately adopted. A member of the PMC will ensure the changes have been captured and recorded. Monthly stakeholder meetings will be open to the entire community, but only members of the PMC have decision making authority. 

```

Comment:
```suggestion
[INSERT FREQUENCY OF STAKEHOLDER INPUT, E.G. MONTHLY SOLICITATIONS] are held to discuss current project status and propose changes to the project roadmap. If stakeholder representatives and sponsors concur with these proposals during the meeting, they will be immediately adopted. A member of the PMC will ensure the changes have been captured and recorded. Stakeholder solicitations will be open to the entire community, but only members of the PMC have decision making authority. 

```

Reply:
Monthly might be overkill for some small projects. Perhaps we suggest the frequency of input as a template item.